Octokit/Helpers/StringExtensions.cs

```diff
@@ -19,6 +19,16 @@ public static bool IsNotBlank(this string value)
         {
             return !string.IsNullOrWhiteSpace(value);
         }
+        
+        public static bool IsNotDefault(this DateTime value)
```

Comment:
Can we move these new methods out to a separate file? They're not exactly string-based, but I'm fine with a single file for these "other" types...

Reply:
I created 2 new files (DateTimeExtensions, and BooleanExtensions.)